### PR TITLE
[backport] Pilz planner merge

### PR DESCRIPTION
**Pull request by martiniil**
### Description

Melodic backport adding the pilz_industrial_motion_planner (#1893)

Tutorial: https://github.com/ros-planning/moveit_tutorials/pull/583

### Checklist
- [x] **Required by CI**: Code is auto formatted using [clang-format](http://moveit.ros.org/documentation/contributing/code)
- [x] Extend the tutorials / documentation [reference](http://moveit.ros.org/documentation/contributing/)
- [x] ~~Document API changes relevant to the user in the [MIGRATION.md](https://github.com/ros-planning/moveit/blob/master/MIGRATION.md) notes~~
- [x] ~~Create tests, which fail without this PR [reference](https://ros-planning.github.io/moveit_tutorials/doc/tests/tests_tutorial.html)~~
- [x] ~~Include a screenshot if changing a GUI~~
- [x] While waiting for someone to review your request, please help review [another open pull request](https://github.com/ros-planning/moveit/pulls) to support the maintainers

[//]: # "You can expect a response from a maintainer within 7 days. If you haven't heard anything by then, feel free to ping the thread. Thank you!"

### CODE REVIEW

**Review by jschleicher — APPROVED** [review]
This cherry-pick brings the pilz_industrial_motion_planner to melodic.

**Comment by v4hn** [comment]
I'm puzzled about this not being merged after *weeks*. This was previously reviewed, passes CI and *only adds functionality* without touching anything else.
Merging.

File: moveit_setup_assistant/templates/moveit_config_pkg_template/launch/move_group.launch [hunk 1]
```diff
@@ -46,13 +47,16 @@
   <!-- Planning Functionality -->
   <include ns="move_group" file="$(find [GENERATED_PACKAGE_NAME])/launch/planning_pipeline.launch.xml">
     <arg name="pipeline" value="$(arg pipeline)" />
+    <param name="capabilities" value="$(arg capabilities)"/>
+    <param name="disable_capabilities" value="$(arg disable_capabilities)"/>
```
Comment (gavanderhoorn) [inline moveit_setup_assistant/templates/moveit_config_pkg_template/launch/move_group.launch:51]: I've just ran into a problem with `roslaunch` complaining that `param` is unexpected here in an `include`. And it's correct.

Should this be `arg` instead? Seems this is a problem on Melodic, which doesn't have #2127, which corrected this.

Comment (gavanderhoorn) [reply moveit_setup_assistant/templates/moveit_config_pkg_template/launch/move_group.launch:51]: #1893 introduced this in `master` ([here](https://github.com/ros-planning/moveit/commit/cf218879dbc23aadf88dd56b8abe7970b7d61030#diff-c9666b8ddf7fb2b57118595db82fca3bb31b167fb07876b124ca6a6f94ba33df)), but that got overwritten by #2127 ([here](https://github.com/ros-planning/moveit/commit/b0a4173c72a9376b1b3a8c63e9bfcc26626184eb#diff-c9666b8ddf7fb2b57118595db82fca3bb31b167fb07876b124ca6a6f94ba33df)).

All MoveIt configuration packages generated by the MSA on Melodic seem to be broken by this.

Comment (gavanderhoorn) [reply moveit_setup_assistant/templates/moveit_config_pkg_template/launch/move_group.launch:51]: An issue has better visibility: https://github.com/ros-planning/moveit/issues/2675.

**Pull request merged**